spec/models/member_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Member do
+  let(:group) { Group.create!(name: 'Design') }
+  let(:member) { Member.create!(name: 'John Doe', email: 'jd@artsymail.com', group_ids: [group.id]) }
+  context '#email' do
+    it 'stores a lowercase version of #email' do
+      expect(member).to be_valid
```

Comment:
This doesn't test what you describe. It checks that the whole record is valid. What you want is to `new` one here that has an email like `jD@foO@bAr` and then test that it was lowercased.